src/main/java/net/minecraftforge/fml/relauncher/ReflectionHelper.java

```diff
@@ -261,7 +344,7 @@ public static Method findMethod(@Nonnull Class<?> clazz, @Nonnull String methodN
             for (int i = 0, length = parameterTypes.length; i < length; i++)
             {
                 desc.append(parameterTypes[i].getName());
-                if (i > length)
+                if (i < length - 1)
```

Comment:
I think it would be best to avoid using indexes, maybe this way:
```java
List<String> parameterNames = Stream.of(parameterTypes).map(Class::getName).collect(Collectors.toList());
desc.append(Joiner.on(", ").join(parameterNames));
```

Reply:
Just fyi, if you go for this, use `Collectors.joining(", ")` rather than collecting to a list.

Reply:
Nice, thanks. I was looking for a stream join method but overlooked `Collectors`.